src/blueprints/tv2/factories/tv2-graphic-action-factory.ts

```diff
@@ -0,0 +1,101 @@
+import { Tv2BlueprintConfiguration } from '../value-objects/tv2-blueprint-configuration'
+import { Action, PieceAction } from '../../../model/entities/action'
+import { PieceInterface } from '../../../model/entities/piece'
+import { PieceType } from '../../../model/enums/piece-type'
+import { Tv2SourceLayer } from '../value-objects/tv2-layers'
+import { TransitionType } from '../../../model/enums/transition-type'
+import { Tv2GraphicsTimelineObjectFactory } from '../value-objects/tv2-graphics-timeline-object-factory'
+import { PieceLifespan } from '../../../model/enums/piece-lifespan'
+import { PieceActionType } from '../../../model/enums/action-type'
+
+export class Tv2GraphicActionFactory {
+  constructor(
+    private readonly graphicsTimelineObjectFactory: Tv2GraphicsTimelineObjectFactory
+  ) { }
+
+  public createGraphicsActions(blueprintConfiguration: Tv2BlueprintConfiguration): Action[] {
+    return [
+      this.createGraphicsThemeOutAction(blueprintConfiguration),
+      this.createOverlayInitializeAction(),
+      this.createContinueGraphicsAction(),
+    ]
+  }
+
+  private createContinueGraphicsAction(): PieceAction {
+    const duration: number = 1000 // Taken from Blueprints
+    const pieceInterface: PieceInterface = {
+      ...this.createDefaultGraphicPieceInterface(),
+      id: 'continueGraphic',
+      name: 'Continue graphics ',
+      duration,
+      tags: [],
+      timelineObjects: [
+        this.graphicsTimelineObjectFactory.createContinueGraphicsTimelineObject(duration)
+      ]
+    }
+    return {
+      id: 'continueGraphicAction',
+      name: 'Continue Graphic',
+      type: PieceActionType.INSERT_PIECE_AS_ON_AIR,
+      data: pieceInterface
+    }
+  }
+
+  // Todo: Question if 'Graphics' should be removed from method name.  
```

Comment:
In this particular case it would make sense to just call it `createThemeOutAction`.

Sidenote: We should really confirm if it is called "Theme out" in english.

Reply:
Name has been updated to `createThemeOutAction`.